README.md

```diff
@@ -81,38 +81,55 @@ structure.
 
 ## Development
 
-To make changes to the playbook theme, you will need to install the
-dependencies:
+To make changes to the playbook theme, you must have the following installed
+on your local machine:
+
+* Ruby >= 2.6
+* Node.js >= 12
+* Go >= 1.13
```

Comment:
Let's use `rbenv`, `nodenv`, and `goenv` to pin these, and document that we've done that.

Reply:
Done! 👍